extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -97,18 +99,25 @@ public void build(ApplicationInfoBuildItem applicationInfo,
                 .collect(Collectors.toList());
 
         Map<String, Object> configAsMap = StreamSupport.stream(config.getPropertyNames().spliterator(), false)
-                .filter(k -> ALLOWED_GENERATOR.equals(generatorName(k)))
+                .filter(k -> ALLOWED_GENERATORS.contains(generatorName(k)))
                 .collect(Collectors.toMap(k -> PROPERTY_PREFIX + k, k -> config.getValue(k, String.class)));
+
         // this is a hack to get kubernetes.registry working because currently it's not supported as is in Dekorate
```

Comment:
Does this comment still apply?

Reply:
I don't remember why it was initially added.

Reply:
I had added it because setting `kubernetes.registry` was not having an effect. I assume this has been fixed now? If so, the comment is no longer needed

Reply:
OK, let's leave it for now